osu.Game/Online/Chat/Message.cs

```diff
@@ -85,6 +95,6 @@ public virtual bool Equals(Message other)
         // ReSharper disable once ImpureMethodCallOnReadonlyValueField
         public override int GetHashCode() => Id.GetHashCode();
 
-        public override string ToString() => $"[{ChannelId}] ({Id}) {Sender}: {Content}";
+        public override string ToString() => $"({(Id?.ToString() ?? "null")}) {Timestamp} {Sender}: {Content}";
```

Comment:
Why was the `ChannelId` removed entirely here?

I always get jittery at `.ToString()` changes, as they may break literally anything anywhere, although this one doesn't look like it would have been used anywhere user-facing.

Reply:
Yeah.. I'm not sure what to do about these scenarios. This particular case is only for debug output, and my rationale for not including the `ChannelId` is that usually messages are grouped by channel already, so it's rarely of concern.

Slightly related, we may want to look at moving to `DebuggerDisplayAttribute` at some point for cases like this, to remove all concern of breakage.